storybook/tsconfig.json

```diff
@@ -1,7 +1,8 @@
 {
   "compilerOptions": {
     "jsx": "react-jsx",
-    "allowSyntheticDefaultImports": true
+    "allowSyntheticDefaultImports": true,
+    "esModuleInterop": true
```

Comment:
I don't think we want this. If you need it for `clsx`, we can import it like this instead: `import { clsx } from 'clsx'`